src/db.rs

```diff
@@ -125,13 +125,20 @@ fn migrate_v0_v1(txn: &Transaction) -> anyhow::Result<()> {
     // drop the old refs table, since the content can be extracted from blocks_v0
     txn.execute_batch("DROP TABLE IF EXISTS refs;")?;
     txn.execute_batch(INIT)?;
+    let mut stmt = txn.prepare("SELECT COUNT(*) FROM blocks_v0")?;
+    let num_blocks: i64 = stmt.query_row(NO_PARAMS, |r| r.get(0))?;
```

Comment:
Not sure if we need a prepared statement for this, since we only execute it once. No harm though.

Reply:
good point, fixed